lib/slack_api.rb

```diff
@@ -12,6 +12,7 @@ def list_channels
   end
 
   def post_message(channel_id, message, options={})
+    return unless channel_id
```

Comment:
i added a `valid_event?` method to master. we should check if the channel_id is present there. ideally by the time we get to post_message we are committed to processing the event.